docs/md/iofogctl_delete_agent.md

```diff
@@ -6,11 +6,9 @@ Delete an Agent
 
 Delete an Agent.
 
-The Agent will be unprovisioned from the Controller within the namespace.
-
-You cannot delete unprovisioned Agents.
-
-The Agent stack will not be uninstalled from the host.
+The Agent will be deleted from the Controller within the namespace.
+The Agent stack will be uninstalled from the host.
+If you wish to not remove the Agetn stack from the host, please use iofogctl detach agent
```

Comment:
Agetn typo

Reply:
Instead of referring to Agent stack on host, can we use "uninstall Agent from host"?